test/test-luatest/smoke_check_test.lua

```diff
@@ -1,10 +1,11 @@
 local t = require('luatest')
-local g = t.group()
 
-local luatest_helpers = require('test.luatest_helpers')
+local server = require('test.luatest_helpers.server')
```

Comment:
I'm not a fun of OOP on Lua, but luatest name classes (modules that acts like classes) from the capital letter and I would keep the naming style uniform.